hw/ip_templates/racl_ctrl/dv/racl_ctrl_sim.core.tpl

```diff
@@ -0,0 +1,32 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: ${instance_vlnv("lowrisc:dv:racl_ctrl_sim:0.1")}
+description: "A racl_ctrl simulation"
+filesets:
+  files_rtl:
+    depend:
+      - ${instance_vlnv("lowrisc:constants:top_racl_pkg")}
+      - ${instance_vlnv("lowrisc:ip:racl_ctrl")}
+
+  files_dv:
+    depend:
+      - lowrisc:dv:racl_ctrl_test
```

Comment:
Is that core file already here?

Reply:
Yep, I think so. Again, this is (intentionally) not templated.